package.json

```diff
@@ -99,6 +99,7 @@
       "x-pack",
       "x-pack/plugins/*",
       "x-pack/legacy/plugins/*",
+      "x-pack/plugins/endpoint",
```

Comment:
is this needed considering the glob two lines above this?

Reply:
perhaps not. will try removing

Reply:
https://github.com/elastic/kibana/pull/53825 should fix

Reply:
@andrewstucki thanks for the catch. removed